stable/osg-hosted-ce/osg-hosted-ce/templates/policy.yaml

```diff
@@ -0,0 +1,40 @@
+{{ if .Values.NetworkPolicy.Enabled }}
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: {{ template "osg-hosted-ce.fullname" . }}
+  labels:
+    app: {{ template "osg-hosted-ce.name" . }}
+    chart: {{ template "osg-hosted-ce.chart" . }}
+    release: {{ .Release.Name }}
+    instance: {{ .Values.Instance | quote }}
+spec:
+  podSelector:
+    matchLabels:
+      app: {{ template "osg-hosted-ce.name" . }}
+      chart: {{ template "osg-hosted-ce.chart" . }}
+      release: {{ .Release.Name }}
+      instance: {{ .Values.Instance | quote }}
+  policyTypes:
+  - Ingress
+  - Egress
+  egress:
+  - to:
+    {{- range .Values.NetworkPolicy.AllowedCIDRs}}
+    - ipBlock:
+        cidr: {{ . }}
+    {{- end }}
```

Comment:
Is using the same IP ranges between ingress and egress the right thing to do?

Reply:
It's kind of just the simplest thing to do.

 We could definitely build a distinction in the chart values between ranges for ingress and ranges for egress, but it does complicate the configuration, and force the user to think about the distinction between traffic that needs to be received by the CE and traffic that needs to be sent by it. 

This requires a little bit of "grid knowledge", but if we set up nice defaults with good README+comments, I think it would be fine to build in the distinction.

What are your thoughts?